### ISSUE TITLE
Error getting Bintray languages

### ISSUE THREAD

**Issue by SimonIT** [Almighty-Alpaca/JetBrains-Discord-Integration#95]
```java
Error getting Bintray languages

java.lang.RuntimeException: 504
	at com.almightyalpaca.jetbrains.plugins.discord.plugin.source.bintray.BintraySource$get$$inlined$suspendCancellableCoroutine$lambda$1.onResponse(BintraySource.kt:196)
	at com.almightyalpaca.jetbrains.plugins.discord.dependencies.okhttp3.internal.connection.RealCall$AsyncCall.run(RealCall.kt:504)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
	Suppressed: java.util.concurrent.ExecutionException: java.lang.RuntimeException: 504
		at java.base/java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:395)
		at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:1999)
		at com.almightyalpaca.jetbrains.plugins.discord.plugin.source.bintray.BintraySource$readLanguages$2$map$2.apply(BintraySource.kt:138)
		at com.almightyalpaca.jetbrains.plugins.discord.plugin.source.bintray.BintraySource$readLanguages$2$map$2.apply(BintraySource.kt:41)
		at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
		at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
		at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1654)
		at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
		at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
		at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913)
		at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
		at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578)
		at com.almightyalpaca.jetbrains.plugins.discord.shared.utils.StreamUtilsKt.toMap(StreamUtils.kt:23)
		at com.almightyalpaca.jetbrains.plugins.discord.plugin.source.bintray.BintraySource$readLanguages$2.invokeSuspend(BintraySource.kt:140)
		at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
		at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:56)
		at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:571)
		at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:738)
		at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:678)
		at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:665)
	[CIRCULAR REFERENCE:java.lang.RuntimeException: 504]
```

Plugin version: 1.3.0
IntelliJ IDEA 2020.1.1 (Ultimate Edition)
Build #IU-201.7223.91, built on April 30, 2020
Runtime version: 11.0.6+8-b765.40 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Windows 10 10.0
GC: ParNew, ConcurrentMarkSweep
Memory: 1631M
Cores: 8

**Comment by Almighty-Alpaca** [comment]
This is currently being worked on

**Comment by Almighty-Alpaca** [comment]
A fix for this was released this morning with version v1.4.0
